build.gradle

```diff
@@ -157,6 +157,7 @@ allprojects {
             // through git history (see "license" section below)
             licenseHeaderFile("${project.rootDir}/licence-header.txt")
 
+            toggleOffOn()
```

Comment:
Why this is necessary?

Reply:
@Theoklitos ^

Reply:
I use it in one test, spotless formatting doesn't really work well for test data params and makes things harder to read

Reply:
I understand the motivation, but I'm not sure it is a good idea to start using this as the idea of introducing the automated code formatting was to remove subjective opinion from the process.

Reply:
I can remove it if you insist, although I'm pretty sure it makes that test code objectively harder to read

Reply:
removed the flag